share/github-backup-utils/ghe-backup-storage

```diff
@@ -117,7 +117,8 @@ fi
 bm_start "$(basename $0) - Storage object sync"
 for file_list in $tempdir/*.rsync; do
   hostname=$(basename $file_list .rsync)
-  storage_user=$(ghe-ssh $ssh_config_file_opt $hostname:$port -- stat -c %U /data/user/storage || echo git)
```

Comment:
`stat` has different format options between GNU and BSD. Use ls+awk which works across OSes. Alternative considered: detect macos and use `gstat` from brew-installed coreutils.